merge_sort.py

```diff
@@ -0,0 +1,13 @@
+from heapq import merge
+
+def merge_sort(arr):
+
+	N = len(arr)
+	if N <= 1:
+		return arr
+
+	# Recursively sort the array
```

Comment:
Thanks for the helpful explanation of your code!

Reply:
It's trivial, you're welcome